workbench-core/authorization/src/dynamicAuthorization/dynamicAuthorizationService.ts

```diff
@@ -229,6 +234,23 @@ export class DynamicAuthorizationService {
     throw new Error('Not implemented');
   }
 
+  /**
+   * Get all identity permissions associated to a specific identity
+   * @param getIdentityPermissionsByIdentityRequest - {@link GetIdentityPermissionsByIdentityRequest}
+   *
+   * @returns - {@link GetIdentityPermissionsByIdentityResponse}
+   */
+  public async getIdentityPermissionsByIdentity(
+    getIdentityPermissionsByIdentityRequest: GetIdentityPermissionsByIdentityRequest
+  ): Promise<GetIdentityPermissionsByIdentityResponse> {
+    getIdentityPermissionsByIdentityRequest = GetIdentityPermissionsByIdentityRequestParser.parse(
```

Comment:
To help delineate when an item has been parsed/validated and when it hasn't, we've been using `validatedRequest`. Not a blocker though.